### DISCUSSION TITLE
视频宽高是否可以动态赋值？

### DISCUSSION THREAD

**Discussion by Flonee** [Tencent/vap#201] [Q&A]
视频宽高是否可以动态赋值？

**Comment by hexleo** [comment]
视频制作完成，宽高就固定了，无法重新赋值

**Reply by Flonee** [reply]
同一页面可以同时播放多个嘛？

**Reply by hexleo** [reply]
可以，但不建议，因为播放越多，性能消耗也越多，需要自己权衡

**Comment by Flonee** [comment]
同一页面引用的时候，共用了一张画布导致多个视频播放的时候重叠在了一起，是不是我设置上有什么问题呢？
web上在视频播放期间拖动改变浏览器窗口大小是不是视频不能能随着窗口一起缩放？

**Reply by hexleo** [reply]
你可以在issue里提这个问题，会有web端的同学给你回答

**Reply by hujinliang** [reply]
> 
1、参数中有个type字段，将这个字段分别设置成不同值可以不共用画布
2、目前是不动态缩放的，要做的话现在可以监听window resize事件，然后用transform来调整大小
